src/components/Launcher.js

```diff
@@ -21,14 +30,21 @@ class Launcher extends Component {
     });
   }
 
+  //call this function to open chat window
+  openChat() {
+    this.setState({
+      active: true
```

Comment:
This is kind of a suboptimal way of doing this. I think we need to say:
Setting `isOpen/isClosed/toggle` manually makes this a `controlled` component in the case of it's active state. In that case, we wouldn't want to to be maintaining an internal active state.

Either the user controls `Launcher.state.active` from above via props, or we do internally via the state. Having both in play at once can cause their app state to be out of sync with the internal state of the component.